x-pack/plugin/ilm/qa/multi-node/src/test/java/org/elasticsearch/xpack/slm/SnapshotLifecycleIT.java

```diff
@@ -388,6 +376,127 @@ public void testSnapshotInProgress() throws Exception {
         }
     }
 
+    @SuppressWarnings("unchecked")
+    public void testRetentionWhileSnapshotInProgress() throws Exception {
+        final String indexName = "test";
+        final String policyName = "test-policy";
+        final String policyName2 = "test-policy2";
+        final String repoId = "my-repo";
+        final String repoId2 = "my-repo2";
```

Comment:
Nitpick: It would be a bit easier to read the test case if instead of `repoId`/`repoId2` it was `slowRepo`/`fastRepo` (same for `policyName`)